docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,325 @@
+
+Please be so kind as to provide `alt` values for all images to save us effort in meeting Accessibility benchmarks. Otherwise, authors should be able to rely on the publishing process to handle technical requirements related to accessibility.
+
+At the same time, the builders of the back end support for these publications will be interested to know of any accessibility-related requirements that are not already adequately addressed.
+
+If publishing a PDF and not simply web resources through the Pages platform, please coordinate and keep in mind requirements in this area for accessible PDFs (PDFa format).
+
+- [ADA.gov resources](https://beta.ada.gov/web-guidance/)
+- [US Web Design System Accessibility page](https://designsystem.digital.gov/documentation/accessibility/)
+- Also search the NIST Intranet for applicable guidance
+
+### Acronyms
+
+The general rule is to spell out acronyms on first use. Keep in mind not all readers are experts in every technology or domain being discussed.
+
+There are cases where the general rule does not apply, because an acronym has become a well-known term of art. <q>HTML</q> does not have to be explained on a page that assumes web technology as context. If an acronym does not actually provide information to anyone in the audience, consider whether it is necessary to explain it. As always, context should be the determining factor. Another example is <q>PR</q> used in reference to git or Github. Similarly this style guide, because it is focused on a web publishing platform, does not explain HTML or CSS, and knowing that HTML expands to <q>Hypertext Markup Language</q> is not important enough to stress, given how easy it is to discover in any case.
+
+### Capitalization
+
+Capitalization is tricky. As a rule, a proper noun is capitalized in English, and hence a reference to a formal object (such as an OSCAL <q>Control</q> object or a Github <q>Issue</q>) should be capitalized, when it has its own name. Both abstractions and things that are not abstract may have names, such as the host of our web presence, which is named <q>Pages</q> (or <q>the NIST Pages site</q>). (Other examples include Avogadro's Number, the Second Law of Thermodynamics, the OSCAL Style Guide, and Markdown, a family of text encoding technologies.)
+
+But even a well-known abstraction such as <q>1 + 1 = 2</q> is not called an <q>Equation</q> (capitalized). To use capitals when referring to something by name is different from instantly or implicitly coining a new technical term out of nothing, by using capitals. To call a Github ticket an <q>Issue</q> is not incorrect if that is what Github calls it; but a ticket in another tracking system is not an <q>Issue</q> (capitalized). In an OSCAL context, even a technical term such as <q>profile resolution</q> should not be capitalized just because is used in a specialized sense.
```

Comment:
At the end of this line, you probably meant to write "because it is used" instead of "because is used".